benchmarks/coremark/Makefile

```diff
@@ -105,4 +105,8 @@ STACKSIZE += $(CONFIG_COREMARK_STACKSIZE)
 
 MAINSRC += core_main.c
 
+# Build with WebAssembly
```

Comment:
```suggestion
# Build with WebAssembly when CONFIG_INTERPRETERS_WAMR is enabled
```

Reply:
Done!

Reply:
Thank you. Other then the CI fail Good to go from me.